blueprints/structural_sections/concrete/reinforced_concrete_sections/covers.py

```diff
@@ -0,0 +1,46 @@
+"""Module with the representation of the covers for cross-sections."""
+
+from dataclasses import dataclass
+
+from blueprints.type_alias import MM
+
+
+@dataclass
+class CoversRectangular:
+    """Representation of the covers of a rectangular cross-section."""
+
+    upper: MM = 50.0
+    right: MM = 50.0
+    lower: MM = 50.0
+    left: MM = 50.0
+
```

Comment:
should it be checked that cover > 0?

Reply:
check #351  (https://github.com/Blueprints-org/blueprints/pull/351/commits/02156116e874764c4214b414497b318554ef74d5)